apps/landingPage/src/components/Demo.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import React from 'react';
+import { Heading, Button } from '@kiwicom/orbit-components/lib/';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import styled from 'styled-components';
+import { Instagram, Facebook } from '@kiwicom/orbit-components/lib/icons/';
+
+import placeholderPhone from '../images/ios-frame.png';
+import {
+  github,
+  tequila,
+  expoLink,
+  documentation,
+  codeKiwi,
+  codeKiwiInstagram,
+  codeKiwiFacebook,
+} from '../../../../linksConfig';
+
+const links = [
+  { title: 'GitHub', link: github },
+  { title: 'Tequila', link: tequila },
+  { title: 'Expo link', link: expoLink },
+  { title: 'Documentation', link: documentation },
+  { title: 'Code.kiwi.com', link: codeKiwi },
+  { title: 'Code.kiwi.com', link: codeKiwiInstagram, icon: <Instagram /> },
+  { title: 'Code.kiwi.com', link: codeKiwiFacebook, icon: <Facebook /> },
+];
+
+export default () => (
+  <Container>
+    <PhoneImage src={placeholderPhone} alt="placeholder" />
+    <ContainerTextRight>
+      <Heading type="title2">Learn more, visit Github</Heading>
+      <Button type="secondary" width={125}>
+        <LinkNoStyle href={github}>Link to Docs</LinkNoStyle>
+      </Button>
+    </ContainerTextRight>
+    <HeadingWrapper>
+      <Heading type="title1">More links to explore</Heading>
+    </HeadingWrapper>
+    <LinksWrapper>
+      {links.map(el => {
```

Comment:
Just cosmetic question.
The `el` is the abbreviation for an element, right? So should be this call `link` or something like that?

Reply:
I was just avoiding using link.link but now i renamed to link.url everywhere